### PR TITLE
Add the location table back in for courses with multiple locations

### PR DESCRIPTION
**Pull request by davidgisbey**
### Context

We're adding back in the locations table for courses with multiple sites. It now should be in the where you will train section instead of the Apply section

### Changes proposed in this pull request

- Add back in the locations table for course with multiple sites and vacancies available

image

### Guidance to review

This sits behind the ucas_only_locations feature flag, as when that is switched on the locations render in the Apply section as before.

We should remove that flag.

### Checklist

- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Review by TobyRet — APPROVED** [review]
LGTM

File: app/views/courses/_about_schools.html.erb [hunk 1]
```diff
@@ -1,17 +1,46 @@
...
+      <% end %>
     <% end %>
-  <% elsif course.program_type == 'scitt_programme' %>
-    <%= render AdviceComponent.new(title: 'Where you will train') do %>
-      <%= render partial: 'courses/placement/scitt', locals: { course: course } %>
+    <% if course.how_school_placements_work.present? %>
+      <%= markdown(course.how_school_placements_work) %>
+    <% end %>
+
+    <% if FeatureFlag.active?(:ucas_only_locations) && course.has_vacancies? && course.site_statuses.map(&:site).uniq.many? %>
```
Comment (paulrobertlloyd) [inline app/views/courses/_about_schools.html.erb:17]: As mentioned.

```suggestion
    <% if FeatureFlag.active?(:ucas_only_locations) && course.site_statuses.map(&:site).uniq.many? %>
```

File: app/views/courses/_about_schools.html.erb [hunk 2]
```diff
@@ -1,17 +1,46 @@
...
+      <h2 class="govuk-heading-m">Locations</h2>
```
Comment (paulrobertlloyd) [inline app/views/courses/_about_schools.html.erb:18]:
```suggestion
      <h3 class="govuk-heading-m">Locations</h3>
```

File: app/views/courses/_about_schools.html.erb [hunk 3]
```diff
@@ -1,17 +1,46 @@
...
+      <p class="govuk-body">You can select one of the following locations when you apply.</p>
+      <table class="govuk-table app-table--vertical-align-middle" data-qa="course__choose_a_training_location">
+        <caption class="govuk-visually-hidden">Choose a training location - List of locations, vacancies and codes</caption>
```
Comment (paulrobertlloyd) [inline app/views/courses/_about_schools.html.erb:21]:
```suggestion
        <caption class="govuk-visually-hidden">List of locations and vacancies</caption>
```

**Review by paulrobertlloyd — CHANGES_REQUESTED** [review]

**Comment by davidgisbey** [comment]
Thanks Paul. Have made the changes.

**Review by paulrobertlloyd — APPROVED** [review]

**Pull request merged**